### ISSUE TITLE
Login/Register links, move to secondary navigation, remove adminbar

### ISSUE THREAD

**Issue by jasmussen** [WordPress/wporg-main-2022#420]
Doing a quick review of navigating across all sections of WordPress.org this morning, and noticing a jump in the top navigation for every page that shows the adminbar. It seems that in every case where the adminbar shows, it exists to surface the login and register links. Here are pages that show the adminbar for this reason.

News:

![news](https://github.com/WordPress/wporg-main-2022/assets/1204802/ba945188-ef3a-4bdd-a590-0f83d149b89d)

Themes:

![themes](https://github.com/WordPress/wporg-main-2022/assets/1204802/7c3d45f1-33eb-4f21-93bf-9e5b25987e23)

Plugins:

![plugins](https://github.com/WordPress/wporg-main-2022/assets/1204802/5885eb03-2d7d-4a1f-aad7-976704979737)

Patterns:

![patterns](https://github.com/WordPress/wporg-main-2022/assets/1204802/883d8696-56e6-49d8-9a1b-0c4965aa6b82)

Learn:

![learn](https://github.com/WordPress/wporg-main-2022/assets/1204802/60206db3-07f9-41cf-aab3-fefd2cd892c6)

Forums:

![forums](https://github.com/WordPress/wporg-main-2022/assets/1204802/45009b26-23f4-43c8-a493-e2333788fc68)

Make (both landing and all P2s):

![make](https://github.com/WordPress/wporg-main-2022/assets/1204802/707231d7-64a3-4cf1-9edf-d5c2f4a0a6df)

Photos:

![photos](https://github.com/WordPress/wporg-main-2022/assets/1204802/67efb6b0-e4d5-4865-ae79-1399c5c97d07)

FFTF:

![fftf](https://github.com/WordPress/wporg-main-2022/assets/1204802/e4699b72-a6b6-4037-9961-431de968f12c)

Also for reference, when you click "Log In" you see this page:

![login](https://github.com/WordPress/wporg-main-2022/assets/1204802/2687c5f3-1b8d-43b7-a16e-e20274b8ca63)

When you click "Register" you see this page:

![register](https://github.com/WordPress/wporg-main-2022/assets/1204802/e8a5544d-6b96-4fe8-a9c2-3f40b55386b6)

In all these cases, those login/register links exist in context of those pages either being editable by contributors, or places where people can submit themes patterns, or otherwise. Those are all page-contextual actions, however, which suggest hierarchically those links should exist in context of the page, rather than in context of _the site_ as they do when they are the first thing on the page. 

Outside of fixing the hierarchy, addressing the jump is especially important on mobile, where those buttons get extra prominence and height:

![mobile news](https://github.com/WordPress/wporg-main-2022/assets/1204802/21d96550-d98c-4227-afd4-6e133b69ab24)

**Suggestions**

* Can we remove the log-in links and adminbar from Five for the Future? It's not clear why it needs to be there at all.
* Move all login and register links into a secondary navigation bar. 

Mockup showing a single unified Log in/Register link:

<img width="1040" alt="login register links" src="https://github.com/WordPress/wporg-main-2022/assets/1204802/ca6c0ff8-d316-4827-97e5-cdd6cda64529">

This single link leans into #241, which puts a "Register" link right on the login page that'll take you there:
<img width="960" alt="login-register-page" src="https://github.com/WordPress/wporg-main-2022/assets/1204802/de80c1b7-1bb2-4b31-93eb-d0bf84aed8e1">

**Comment by jasmussen** [comment]
Ah good note. To clarify, if you _are logged in_ it's okay to show the adminbar across all sections. And in fact ideally we show it across _all_ sections if you're logged in, so you don't get an inverse jump in that case of the adminbar disappearing.

**Comment by tobifjellner** [comment]
Is showing menu items like "my favorites" for not logged-in visitors desirable?

**Comment by jasmussen** [comment]
> Is showing menu items like "my favorites" for not logged-in visitors desirable?

I'd think so, yes, it makes clear a benefit you get from logging in. Besides, clicking Favorites if logged in, would take you to a screen prompting you to do that.

**Comment by ndiego** [comment]
I wonder if we could explore something similar to the [Amazon](https://www.amazon.com/) experience. 

<img width="493" alt="image" src="https://github.com/user-attachments/assets/665add2e-f7dc-4293-952b-b2373a0ced1d">

The login/register is combined with more info revealed in a dropdown, and it's placed in the primary header. This is tricky with the Navigation block, but might be worth exploring.

**Comment by jasmussen** [comment]
I've updated this issue based on a fresh audit of the WordPress.org site after the recent string of style refreshes, then collaborated with @fcoveram to leverage the new secondary navigation bar for a simple and claer sign-in link that's contextual to the pages that need it.

It's a big win that improves an increasingly frustrating aspect of the new experience, so I added a high priority to it.

I also opened #490 to make the logo bigger, which pairs well with this change now that we're in the same space, code-wise! Penny for your thoughts?

**Comment by jasmussen** [comment]
Good point. "Log in" can work equally well. The motivation for "Sign in" was to simply emphasize the link a bit more.
